### ISSUE TITLE
Tagger complains - some input tokens are not in the dictionary (despite add_unk=True)  - Still a problem

### ISSUE THREAD

**Issue by None-Such** [flairNLP/flair#2879]
As per **#2407** (https://github.com/flairNLP/flair/issues/2407) the Tagger complains that some input tokens are not in the dictionary (despite add_unk=True).

This can easily be recreated by having a tag exist in your Dev or Test set that does NOT exist in your Training set. Which will raise the same error with no indication of what the real problem is.
